scripts/health/common.py

```diff
@@ -0,0 +1,40 @@
+""" """
+import pandas as pd
+import requests
+
+
+def query_who(code: str):
+    """Query the WHO website for a given code.
+    To be replaced in bblocks
+    """
+
+    URL = 'https://ghoapi.azureedge.net/api/'
+
+    request = requests.get(URL + code)
+    data = request.json()
+    df = pd.DataFrame.from_records(data['value'])
+
+    return df
+
+
+def get_malaria_data() -> dict:
+    """Extract and clean malaria data for overview chart"""
+
+    malaria_dict = (query_who('MALARIA_EST_DEATHS')
+        .loc[lambda d: d.SpatialDim.isin(['GLOBAL', 'AFR']), ['SpatialDim', 'TimeDim', 'NumericValue']]
+        .sort_values('TimeDim')
+        .groupby(['SpatialDim'], as_index=False)
+        .last()
+        .assign(NumericValue=lambda d: pd.to_numeric(d.NumericValue))
+        .pivot(index='TimeDim', columns='SpatialDim', values='NumericValue')
+        .reset_index()
+        .assign(malaria_rest_of_world_total=lambda d: d['GLOBAL'] - d['AFR'])
+        .assign(malaria_africa_proportion=lambda d: (d['AFR'] / d['GLOBAL']) * 100)
+        .assign(malaria_rest_of_world_proportion=lambda d: (d['malaria_rest_of_world_total'] / d['GLOBAL']) * 100)
```

Comment:
you can assign multiple columns in the same assign call by separating with a comma. 

```python
.assign(
    malaria_rest_of_world_total=lambda d: d['GLOBAL'] - d['AFR'],
    malaria_africa_proportion=lambda d: (d['AFR'] / d['GLOBAL']) * 100),
    malaria_rest_of_world_proportion=lambda d: 100-d.malaria_africa_proportion
)
```